src/main/api/2/studio-api-2.yaml

```diff
@@ -4085,6 +4085,39 @@ paths:
 #        '500':
 #          $ref: '#/components/responses/InternalServerError'
 
+
+  /translation/config:
```

Comment:
Sorry, just noticed this: perhaps get_config could have better semantics (or retrieve_config or similar). The name config sounds like is a verb or so. Makes me think this is the service to set a config and possibly would be a post.

`get_config` would also better match the operation ID and other API2 items (e.g. content/get_delete_package).

I do like the omission of "translation" within the operation name as we're already in that namespace (i.e. get_config instead of get_translation_config)

Reply:
Sure I'll update to follow the existing pattern, but I think we should consider moving towards a more standard REST API where the URL identifies a resource and the operation is defined by the HTTP method:

```
GET  /translation/config -> retrieves the config
POST /translation/config -> updates the config
```

Reply:
I've always been in favour of that too. 

I believe at the early stages of API2 that was considered; don't remember the reason why we ended up steering away from that.